posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -315,17 +330,17 @@ config.getOptionalValues("ports", Integer.class) // returns "Optional[[9080, 908
 ----
 @Inject
 @ConfigProperty(name = "server.port")
-OptionalInt optionalServerPort; // returns “9080"
+OptionalInt optionalServerPort; // Returns: OptionalInt[9080]
 ----
 
 [#Incompatibility-changes]
-== Heads up! Incompatibility Changes
+== Heads Up! Incompatibility Changes
 If you move up from MicroProfile Config 1.x to 2.0, please take care of the following incompatible changes:
 
-* link:https://javadoc.io/static/org.eclipse.microprofile/microprofile/4.0.1/org/eclipse/microprofile/config/Config.html#getPropertyNames--[ConfigSource.getPropertyNames()] is no longer a `default` method. Any implementations of a ConfigSource must implement this method.
+* link:https://javadoc.io/static/org.eclipse.microprofile/microprofile/4.0.1/org/eclipse/microprofile/config/Config.html#getPropertyNames[ConfigSource.getPropertyNames()] is no longer a `default` method. Any implementations of a ConfigSource must implement this method.
 * Previous versions of MP Config would not evaluate Property Expressions. As such, previous working configuration may behave differently (if the configuration contains values with Property Expressions syntax, e.g. `${var.name}`). Property Expressions can be disabled by setting the property `mp.config.property.expressions.enabled` with the value of `false`.
```

Comment:
Change,
"Previous versions of MP Config would not evaluate Property Expressions. As such, previous working configuration may behave differently (if the configuration contains values with Property Expressions syntax, e.g. `${var.name}`)."
to,
"Previous versions of MP Config do'ot evaluate Property Expressions. As such, previous working configuration might behave differently (if the configuration contains values with Property Expressions syntax, e.g. `${var.name}`)."